centos.org/pipelines/foreman-discovery-image-build.groovy

```diff
@@ -32,7 +33,7 @@ pipeline {
             steps {
                 script {
                     duffy_ssh("git clone https://github.com/theforeman/foreman-discovery-image/ --branch ${env.branch}", 'duffy_box', './')
-                    duffy_ssh("cd foreman-discovery-image/aux/vagrant-build/ && vagrant up fdi-builder", 'duffy_box', './')
+                    duffy_ssh("cd foreman-discovery-image/aux/vagrant-build/ && vagrant --repo-owner=${env.repo_owner} --branch=${env.branch} --proxy-repo=${env.proxy_repository} up fdi-builder", 'duffy_box', './')
```

Comment:
How does this work? I'm not aware of vagrant accepting custom parameters.

Reply:
See here: https://github.com/theforeman/foreman-discovery-image/blob/cos8-novirt/aux/vagrant-build/Vagrantfile

Reply:
Ok I reverted it back to ENV variables: https://github.com/theforeman/foreman-discovery-image/commit/1b46f13bea3b4d80e2c3929aabbf2e64d9aa0e7b